CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.6.1 (2020-09-08)
```

Comment:
you didn't change package.json version number, so this won't be released. replace the date by `not released yet`

Reply:
should I rather change the package json version?